lib/kubeclient/common.rb

```diff
@@ -271,7 +252,7 @@ def watch_entities(resource_name, options = {})
     #
     #   Default response type will return a collection RecursiveOpenStruct
     #   (:ros) objects, unless `:as` is passed with `:raw`.
-    def get_entities(entity_type, klass, resource_name, options = {})
```

Comment:
Are `get_entities`, `get_entity` public APIs?  :+1: on this change, but if yes it should be documented.

Reply:
They are not, we agreed to add a comment separately in the README.md saying any usage 
not documented in the README.md is not supported.